### ISSUE TITLE
Error kW display instead of Watt

### ISSUE THREAD

**Issue by fcoutellier** [LucasBrazi06/ev-dashboard#1778]
You see a 500kW between 0 kW and 1kW.
![image.png](https://images.zenhubusercontent.com/5e411e0517fa45e1af2c53aa/4e1fcfbf-4500-4be8-96f3-e4fe8911337b)

**Comment by OliveGerste** [comment]
I checked all the diagrams in the application. The Problem seems to be solved.
The only thing I noticed is, that in the site area consumption no unit on the y-axis is displayed (like in the other diagrams).
![image](https://user-images.githubusercontent.com/33114317/94119513-4fa05300-fe4f-11ea-8004-b725f4fd6e65.png)
![image](https://user-images.githubusercontent.com/33114317/94119702-8e360d80-fe4f-11ea-98c9-e2d3e15ded40.png)
